src/raftstore/store/peer.rs

```diff
@@ -227,6 +228,7 @@ pub struct Peer {
 
     pub written_bytes: u64,
     pub written_keys: u64,
+    pub latestn_written_bytes: LatestN,
```

Comment:
the bytes will be only reset when reported to PD, I think it is ok and more simple.